builtin/game/auth.lua

```diff
@@ -199,3 +199,19 @@ core.register_on_joinplayer(function(player)
 	record_login(player:get_player_name())
 end)
 
+core.register_on_prejoinplayer(function(name, ip)
+	local auth = core.auth_table
+	if auth[name] ~= nil then
+		return
+	end
+
+	local name_lower = name:lower()
+	for k,v in pairs(auth) do
```

Comment:
Just a nitpick. But `for k in` avoids the linter warning of an unused variable `v`.